### PR TITLE
feat: add missing TWAP events and some fortune cookies

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

TWAP analytic events from this [document](https://docs.google.com/document/d/1qEY9OXeRsY7AGnpJRoPdJPQr1kqoKX3BUOeAfbW4Uhg/edit)

Events

- [x] When you are on advanced-orders widget and you connect a wallet (compatible, non-compatible, safe-that-could-be-converted)
- [x] When user checks "Modify Safes handler" checkbox
- [ ] Click on read more (These components are still not implemented)
- [x] Open on advanced orders tab

These events will happen only when the user still hasn't set fallback handler and now is trying to
- [x] TWAP with conversion initiated (When users clicks on "Review TWAP order" button)
- [x] TWAP with conversion posted (When he confirms in the modal)
- [x] TWAP with conversion signed (When he signs in safe)
- [x] TWAP with conversion rejected (When he rejects signing)

Some COW fortune events

- [x] Share COW Fortune on Twitter
- [x] Open COW Fortune 

Deprecated
- [x] ~~Click on “Change connected wallet” button~~ **Deprecated**
- [x] ~~Click on “Setup supported Safe” button~~ **Deprecated**

To test
1. turn on GA debugger
2. test different events

### CODE REVIEW

File: src/legacy/components/analytics/index.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ if (typeof GOOGLE_ANALYTICS_ID === 'string') {
       storage: 'none',
       storeGac: false,
       clientId: storedClientId ?? undefined,
+      cookieFlags: 'SameSite=None; Secure',
```
Comment (nenadV91) [inline src/legacy/components/analytics/index.ts:44]: This was fix for Safe App not sending GA events

File: src/modules/application/containers/TradeWidgetLinks/index.tsx [hunk 2]
```diff
@@ -15,12 +17,18 @@ interface MenuItemConfig {
...
 
 const menuItems: MenuItemConfig[] = [
   { route: Routes.SWAP, label: 'Swap' },
   { route: Routes.LIMIT_ORDER, label: 'Limit' },
-  { route: Routes.ADVANCED_ORDERS, label: 'Advanced', featureGuard: 'advancedOrdersEnabled' },
+  {
+    route: Routes.ADVANCED_ORDERS,
+    label: 'Advanced',
+    featureGuard: 'advancedOrdersEnabled',
+    onClick: clickAdvancedOrdersTabAnalytics,
```
Comment (shoom3301) [inline src/modules/application/containers/TradeWidgetLinks/index.tsx:30]: You can replace it by adding code to `TwapFormWidget`:

```
useEffect(() => {
    sendEvent({
        category: Category.TWAP,
        action: 'Click Advanced Orders Tab',
    })
}, [])
```

**Review by shoom3301 — APPROVED** [review]
Good! Thank you!
I've added a small fix to prevent falsy triggers of `safe-that-could-be-converted`

File: src/modules/application/containers/TradeWidgetLinks/index.tsx (hunk 2 above)
Comment (nenadV91) [reply src/modules/application/containers/TradeWidgetLinks/index.tsx]: Good suggestion, updated 👍 Now the action will be `Open Advanced Orders Tab`

**Review by alfetopito — APPROVED** [review]
Looks good, just a minor comment

File: src/legacy/components/analytics/events/twapEvents.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+type TwapWalletCompatibility = 'non-compatible' | 'compatible' | 'safe-that-could-be-converted'
+export function twapWalletCompatibilityAnalytics(action: TwapWalletCompatibility) {
+  sendEvent({
+    category: Category.TWAP,
+    action: `TWAP wallet compatibility ${action}`,
+  })
+}
+
+export function modifySafeHandlerAnalytics(action: boolean) {
```
[the remark this answers is not in the document]
Comment (nenadV91) [reply src/legacy/components/analytics/events/twapEvents.ts]: Makes sense, went with the second option 👍

**Review by elena-zh — APPROVED** [review]
All events (besides the one that is not yet implemented) LGTM

**Pull request merged**